packages/block-library/src/index.js

```diff
@@ -171,5 +171,7 @@ export const __experimentalRegisterExperimentalCoreBlocks = process.env.GUTENBER
 		__experimentalEnableLegacyWidgetBlock ? legacyWidget : null,
 		__experimentalEnableMenuBlock ? navigationMenu : null,
 		__experimentalEnableMenuBlock ? navigationMenuItem : null,
+		__experimentalEnableLegacyWidgetBlock ? socialLink : null,
+		__experimentalEnableLegacyWidgetBlock ? socialLinks : null,
```

Comment:
Thanks for the PR @nicolad 

Maybe we can just use `process.env.GUTENBERG_PHASE === 2 `  as a test instead of `__experimentalEnableLegacyWidgetBlock`  because If I'm not mistaken, this will ensure the block is always loaded in the plugin but not included in Core.

Can you confirm @talldan ?

Reply:
Thanks @youknowriad, updated.

Reply:
Looks good 👍